Sources/Commands/SwiftTestTool.swift

```diff
@@ -178,6 +199,17 @@ public struct SwiftTestTool: SwiftCommand {
             throw ExitCode.failure
         }
 
+        let buildParameters = try swiftTool.buildParametersForTest(options: self.options, sharedOptions: self.sharedOptions)
+
+        // Remove test output from prior runs and validate priors.
+        if self.options.enableExperimentalTestOutput {
+            if !buildParameters.targetTriple.isDarwin() {
+                swiftTool.observabilityScope.emit(error: "The experimental test output feature is only available on Darwin.")
```

Comment:
I _believe_ we can do the same thing in the generated "runner" code for none-darwin, but probably fine to do as a second step

Reply:
Yep, I think so.